CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.26](https://github.com/brave/brave-ios/releases/tag/v1.26)
+
+- Added support for Auto-Close Tabs. ([#1514](https://github.com/brave/brave-ios/issues/1514))
+- Added ability to disable long press to add media into playlist via the “Playlist” settings. ([#3670](https://github.com/brave/brave-ios/issues/3670))
+- Added a new tab tutorial page on startup for Japan region. ([#3389](https://github.com/brave/brave-ios/issues/3389))
+- Updated settings menu to display scrollable modal which will allow for more dynamic actions. ([#3491](https://github.com/brave/brave-ios/issues/3491))
+- Updated Startpage icon under onboarding and search engine settings. ([#3584](https://github.com/brave/brave-ios/issues/3584))
+- [Security] Fixed cookie persistence via download request in private mode as reported on Bugzilla by Muneaki Nishimura. ([#2984](https://github.com/brave/brave-ios/issues/2984))
+- [Security] Fixed cross-site scripting issue in RSS feed as reported on HackerOne by Muneaki Nishimura. ([#3630](https://github.com/brave/brave-ios/issues/3630))
+- Improved readability of text under “Brave Shields & Privacy” settings by reducing font weight. ([#3180](https://github.com/brave/brave-ios/issues/3180))
+- Fixed playlist videos appearing under “Downloaded Videos” via iOS storage settings. ([#3633](https://github.com/brave/brave-ios/issues/3633))
+- Fixed https://www.listennotes.com not being recognized by Playlist. ([#3726](https://github.com/brave/brave-ios/issues/3726))
+- Fixed “Add to Brave Playlist” toast not being displayed on https://odysee.com. ([#3663](https://github.com/brave/brave-ios/issues/3663))
```

Comment:
Should be `Added playlist support for Odysee.com` . This is a new site that we started supporting not fixing an existing one.

Reply:
Done 👍